types/react-big-calendar/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for react-big-calendar 0.17.0
+// Type definitions for react-big-calendar 0.18.0
```

Comment:
Right now it must be the case that our lint rules aren't active or else it would have complained about the patch version being present. Can you fix up the version to be `0.18` and also enable the appropriate lint rules?